HerPublicWebsite.BusinessLogic/Models/Enums/QuestionFlowStep.cs

```diff
@@ -9,4 +9,5 @@ public enum QuestionFlowStep
     Address,
     SelectAddress,
     GasBoiler,
+    ManualAddress,

```

Comment:
Minor thing, but it will be easier for future readers of the code to understand the flow if we keep this enum in flow order (where possible) so I'd move this above `GasBoiler`